front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -141,25 +139,20 @@ const CollapsibleContainer = ({
           <ChevronIcon name="chevron-right" />
         </TitleButton>
       </Title>
-
       <Box
         role={useRegionRole ? 'region' : undefined}
         aria-live="polite"
         id={`collapsed-section-${uuid}`}
         aria-labelledby={`collapse-container-title-${uuid}`}
-        hidden={!isExpanded}
       >
         <CSSTransition
           in={isExpanded}
-          timeout={1500}
+          timeout={ANIMATION_TIMEOUT}
           mountOnEnter={true}
           unmountOnExit={true}
-          exit={false}
           classNames={`expanded`}
         >
-          <CollapseContainer aria-live="polite">
-            <Box>{children}</Box>
```

Comment:
I don't think Box is necessary. Or do you remember why it's here specifically?

Reply:
Ah you're right - we can get rid of it here :+1: